xtas/tasks/_polarity.py

```diff
@@ -99,7 +99,7 @@ def classify(doc):
             MODEL = load(model_path)
         except (IOError, OSError) as e:
             if e.errno == errno.ENOENT:
-                MODEL = train()
```

Comment:
train() is never called with param_search=True, except maybe from the outside, and then you have to go and explicitly update MODEL from the outside as well? That doesn't seem like a nice modular interface.